sourcemap/cache.go

```diff
@@ -28,12 +28,12 @@ func newCache(expiration time.Duration) (*cache, error) {
 
 func (c *cache) add(id Id, consumer *sourcemap.Consumer) {
 	c.goca.Set(id.Key(), consumer, gocache.DefaultExpiration)
-	logp.Debug("sourcemap", "Added id %v. Cache now has %v entries.", id.Key(), c.goca.ItemCount())
+	logp.NewLogger("sourcemap").Debugf("Added id %v. Cache now has %v entries.", id.Key(), c.goca.ItemCount())
```

Comment:
here also it might make sense to have a global logger for this file

Reply:
Beats documentation warns against using `logp.NewLogger` globally.
 
Can you have a look at the question i wrote you in your logp PR? I don't know what to do with that case (https://github.com/elastic/apm-server/blob/master/beater/handlers.go#L388-L393).

Reply:
ah, didn't see that 👍

Reply:
let's make "sourcemap" a constant instead?

Reply:
added here e305b844f3f52fcefa646e21c4abbb9cb365a8ce